src/docs/concepts-users-and-authentication/users-in-flex/index.md

```diff
@@ -0,0 +1,160 @@
+other than that customers are not required to enter further information
+to use the marketplace. When using the Stripe default payment
+integration, customers can save a payment method, however the payment
+method information is saved in Stripe and not directly in Flex.
+
+Within a transaction, customers can initiate or transition transactions
+specified for the customer.
+
+### Provider
+
+When a user creates a listing and someone else books or purchases that
+listing, the user becomes the **provider** of the transaction in
+question. In marketplaces with payments, the provider is the transaction
+party who receives the listing price.
+
+This means that if the payment happens within the marketplace, the
+provider will need to verify their identity to the payment gateway to
+adhere to the payment processor's
+[Know Your Customer requirements](https://en.wikipedia.org/wiki/Know_your_customer).
+
+Within a transaction, providers can initiate or transition transactions
+specified for the provider.
+
+### Transaction related roles: operator and system
+
+A **marketplace operator** is not a user in the marketplace – they
+cannot sign in to the marketplace with the same credentials they use to
+sign in to Flex Console. The operator can, however, take actions on the
+marketplace through Flex Console or Integration API, when those actions
+are defined for the operator. Operators cannot participate in the
+messaging between customer and provider within the transaction.
+
+A transaction transition is performed by the **system** if it is
+scheduled to happen automatically.
+
+## User access
+
+Flex marketplace listings can be viewed by anyone by default, whether
+they are registered users or not. If a user wants to start a transaction
+or create a listing, they need to be registered to Flex.
+
+### Registered marketplace users
+
+Listings can only be updated by their author, i.e. the registered user
+who originally created the listing. Operators can create listings for a
+registered user, and update existing listings.
+
+Beyond that, Flex does not have different levels of user access within
+the marketplace. Operators who want to create more complex user
+hierarchies will need to think about the levels of user access they want
+each custom role to have and potentially use a custom backend solution
+to complement Flex default user management. If you are contemplating
+creating a user hierarchy in your Flex marketplace, contact
+[Flex Support](mailto:flex-support@sharetribe.com) and let us know your
+use case – we're happy to help you figure out a suitable solution!
+
+### Login as user
+
+Flex has a feature through which operators can log in to their
+marketplace as a registered user and take limited actions on their
+behalf. When using the Login as user feature, operators cannot initiate
+or transition transactions or modify the user's payout information.
+However, they can e.g. create and update listings on behalf of the user.
+
+### Integration API
```

Comment:
I'd maybe leave this section out? Or shift the point towards authentication and that with the user access token you can only access certain Marketplace API endpoints and not the Integration API